frontend/pages/viewer-registration/password-aid/password-aid.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { MouseEvent, useState } from 'react' 
+import styles from './password-aid.module.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { passwordToggleViews } from '../../../models'
+
+interface PasswordAidProps { id: string, onDisplayChange: any }
+export default function PasswordAid({ id, onDisplayChange }: PasswordAidProps) {  
+  const { passwordAid, passwordToggle } = styles
+  const { showView, hideView } = passwordToggleViews
+
+  const [{ icon, isHidden, text }, setPasswordView] = useState(showView)
+  
+  function handlePasswordView($event: MouseEvent<HTMLButtonElement>): void {
```

Comment:
Also thought we were trying to avoid typing everything and were only gonig to type the same data from the backend? This is just my dislike of typescript needing to type everything where I have spent too much time debugging this rather than just getting actual work in.

Reply:
I think we might need to override some typescript settings to not do that - it was throwing an error without a type